cg/meta/invoice.py

```diff
@@ -34,15 +34,10 @@ def prepare_contact_info(self, costcenter):
         )
 
         customer = self.db.customer("cust999") if costcenter.lower() == "kth" else self.customer_obj
-        user = customer.invoice_contact
-
-        if not user:
-            self.log.append(msg)
-            return None
 
         contact = {
-            "name": user.name,
-            "email": user.email,
+            "name": customer.name,
```

Comment:
this is not exactly the same name as the user name, will it be alright anyways?

Reply:
Okej. How do we get the exact same?

Reply:
How important is it to have the contacts name?

Reply:
The name of the invoice contact is important to get onto the invoice

Reply:
I think we need to add another field for the name, I'll look into it

Reply:
Yes. The user name is the Contacty person in the invoice. Using customer.name will not be correct, but there is no alternative atm. Since VW needs to send invoices today we must merge this pr now. If you are not solving the new customer name field in the database before lunch @patrikgrenfeldt? Otherwise Its not a problem to for WV to fix it manually and you can add the new field in a separate pr @patrikgrenfeldt.